src/snowflake/snowpark/functions.py

```diff
@@ -7131,6 +7132,8 @@ def udf(
             also be specified in the external access integration and the keys are strings used to
             retrieve the secrets using secret API.
         immutable: Whether the UDF result is deterministic or not for the same input.
+        comment: Adds a comment or overwrites an existing comment for an existing object. See
```

Comment:
We don't generate `alter` sql so we don't have the "overwrites" scenario?

Reply:
So you mean just remove "overwrite text" here?

Reply:
Yes. Here and other places.